docs/getting_started/index.rst

```diff
@@ -41,7 +41,7 @@ Inside your ``mysite`` folder, we now just run the setup steps necessary for any
 
 Your site is now accessible at ``http://localhost:8000``, with the admin backend available at ``http://localhost:8000/admin/``.
 
-This will set you up with a new standalone Wagtail project. If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
+This will set you up with a new stand-alone Wagtail project. If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
```

Comment:
Isn’t `standalone` the standard nowadays?

Reply:
Google's dictionary auto-corrects this to stand-alone so not sure

Reply:
OK, so choose whichever seems more natural to you, you’re the native speaker here :p

Reply:
Not actually that great at English though! But I'm leaning towards "stand-alone" as they are two separate words really...

Any opinions @tomdyson @gasman?

Reply:
Google results seem to lean towards "stand-alone" too, so let's go with that.

This surprises me a bit - I'm sure I've seen "standalone" used more often, to the point that hyphenating it feels faintly archaic, like writing "to-day" instead of "today". But fair enough!